maintainer/channels.py

```diff
@@ -131,15 +120,3 @@ def list_of_children(self):
         return child_list
 
 
-def main():
-    '''
-    Main is used only in order to debug and show basic functionality of this
-    Module.
-    '''
-    chan = ChannelsMaintanance()
```

Comment:
Is it valid name for the object? Is it meaningful?

Reply:
fixed